netebpfext/sys/net_ebpf_ext_drv.c

```diff
@@ -158,6 +167,23 @@ DriverEntry(_In_ DRIVER_OBJECT* driver_object, _In_ UNICODE_STRING* registry_pat
 
     // Request NX Non-Paged Pool when available
     ExInitializeDriverRuntime(DrvRtPoolNxOptIn);
+
+    status = ebpf_result_to_ntstatus(ebpf_platform_initiate());
```

Comment:
Why wasn't this call required before?

Reply:
I am re-using our hash table implementation. This pulls in a couple of other dependencies, notably platform as well as epoch into netebpfext.